### PR TITLE
Readme badges improvements

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
Not sure yet why codecov says we have 0% coverage in the badge :sweat_smile:

**Review by goetas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Doctrine Migrations
 
-[![Build Status](https://travis-ci.org/doctrine/migrations.svg)](https://travis-ci.org/doctrine/migrations)
-[![Code Coverage](https://codecov.io/gh/doctrine/migrations/branch/3.1.x/graph/badge.svg)](https://codecov.io/gh/doctrine/migrations/branch/3.1.x)
+[![Build Status](https://github.com/doctrine/migrations/workflows/Continuous%20Integration/badge.svg?label=build)](https://github.com/doctrine/migrations/actions)
+[![Code Coverage](https://codecov.io/gh/doctrine/migrations/branch/3.0.x/graph/badge.svg)](https://codecov.io/gh/doctrine/migrations/branch/3.0.x)
 [![Packagist Downloads](https://img.shields.io/packagist/dm/doctrine/migrations)](https://packagist.org/packages/doctrine/migrations)
 [![Packagist Version](https://img.shields.io/packagist/v/doctrine/migrations)](https://packagist.org/packages/doctrine/migrations)
-[![GitHub license](https://img.shields.io/github/license/doctrine/migrations)](https://github.com/doctrine/migrations/blob/3.1.x/LICENSE)
+[![GitHub license](https://img.shields.io/github/license/doctrine/migrations)](https://github.com/doctrine/migrations/blob/3.0.x/LICENSE)
```
Comment (stof) [inline README.md:7]: you could use a relative link `(LICENSE)` here. Github will resolve it (and Packagist too), and that will avoid having to update it in each branch.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Doctrine Migrations
 
-[![Build Status](https://travis-ci.org/doctrine/migrations.svg)](https://travis-ci.org/doctrine/migrations)
-[![Code Coverage](https://codecov.io/gh/doctrine/migrations/branch/3.1.x/graph/badge.svg)](https://codecov.io/gh/doctrine/migrations/branch/3.1.x)
+[![Build Status](https://github.com/doctrine/migrations/workflows/Continuous%20Integration/badge.svg?label=build)](https://github.com/doctrine/migrations/actions)
```
Comment (stof) [inline README.md:3]: the `label` parameter does not exist in the github actions badge URL

**Comment by greg0ire** [comment]
@stof thanks, addressed!

**Pull request merged**